docs/operation/enable_ip_ip.rst

```diff
@@ -0,0 +1,17 @@
+.. _enable IP-in-IP:
+
+Enable IP-in-IP encapsulation
+=============================
+
+.. _IP-in-IP: https://en.wikipedia.org/wiki/IP_in_IP
+
+By default Calico in MetalK8s is configured to use IP-in-IP_ encapsulation only
```

Comment:
Calico would benefit from an explanation in the glossary + external link. Can become a tracking issue if you prefer.